docs/docs/reference/release-notes/v3.0/index.md

```diff
@@ -1,6 +1,7 @@
 ---
 date: "2021-03-02"
 version: "3.0.0"
+title: "v3 Release Notes"
```

Comment:
Not strong feeling about this, but seemed inconsistent?
```suggestion
title: "v3.0 Release Notes"
```

Reply:
We also use `v2 Release Notes` so it'll be consistent with that